script.js

```diff
@@ -0,0 +1,97 @@
+let valueOfInput, description, input, temp, longitudeData, latitudeData;
+console.log("hello");
+const apiKey = "82005d27a116c2880c8f0fcb866998a0";
+const api = () => {
+  valueOfInput = document.getElementById("inputData").value;
+  console.log("inpuuuuuuuuuut", valueOfInput);
+  document.getElementById("icon").src = "loader.gif";
+  fetch(
+    `https://api.openweathermap.org/data/2.5/weather?q=${valueOfInput}&appid=82005d27a116c2880c8f0fcb866998a0`
```

Comment:
wy are you not using apiKey when doing fetch, you written it directly it into the URL @Rishab-innovative